pyedflib/_extensions/_pyedflib.pyx

```diff
@@ -554,9 +556,10 @@ def set_samples_per_record(handle, edfsignal, smp_per_record ):
     """
     int set_samples_per_record(int handle, int edfsignal, int smp_per_record )
 
-    sets how many samples are in the record for this signal.
-    this is not the sampling frequency (Hz), (which is calculated by
-    by smp_per_record/record_duration).
+    sets how many samples are in each record for this signal.
+    this is not the sampling frequency (Hz), unless record_duration=1
+    The sample frequency is calculated by smp_per_record/record_duration.
+    The function call to the C library is therefore slightly mislabeled
     """
```

Comment:
If this is supposed to be a comment, I suggest using an actual comment instead of a string.